eclipse-scout-core/src/desktop/outline/Outline.ts

```diff
@@ -207,30 +210,36 @@ export class Outline extends Tree implements DisplayParent, OutlineModel {
     return scout.create(nodeModel as FullModelOf<Page>);
   }
 
-  protected override _createKeyStrokeContext(): KeyStrokeContext {
-    return new OutlineKeyStrokeContext(this);
-  }
-
   protected override _filterMenus(argMenus: Menu[], destination: MenuDestinations, onlyVisible?: boolean, enableDisableKeyStrokes?: boolean): Menu[] {
     // show no context menus
     return [];
   }
 
   protected override _initTreeKeyStrokeContext() {
+    super._initTreeKeyStrokeContext();
+
     let modifierBitMask = keyStrokeModifier.CTRL | keyStrokeModifier.SHIFT; // NOSONAR
 
-    this.keyStrokeContext.registerKeyStrokes([
+    let outlineKeyStrokeContext = new OutlineKeyStrokeContext(this);
+    outlineKeyStrokeContext.registerKeyStrokes([
```

Comment:
Why do we use two separate keystroke contexts and don't just use the OutlineKeyStrokeContext for everything?

Reply:
OutlineKeyStrokeContenst has a different scope so these keystrokes can be used from anywhere not only if the outline is focused